app/api/i18n/specs/__snapshots__/routes.spec.js.snap

```diff
@@ -19,98 +19,123 @@ Object {
 
```

Comment:
i have updated the snapshot instead of removing it entirely. That's because I have only refactored the parts of the code that has the new endpoint and did not touch `POST /translations` endpoint. I propose we do the refactor in another issue